apps/graphql/src/apps/itinerary/types/outputs/ItineraryValidity.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+import { GraphQLObjectType, GraphQLBoolean } from 'graphql';
+
+import GraphQLItinerary from './Itinerary';
+
+export default new GraphQLObjectType({
+  name: 'ItineraryValidity',
+  fields: {
+    itineraryChecked: {
+      type: GraphQLBoolean,
+      description: 'All segments successfully checked',
+    },
+    itineraryValid: {
+      type: GraphQLBoolean,
+      description:
+        'Itinerary is valid (exists and is not sold-out or cancelled)',
+    },
+    itinerary: { type: GraphQLItinerary, description: 'Actual itinerary data' },
```

Comment:
What is your motivation to return also itinerary data here?

Reply:
Because we need it to fill the itinerary detail in header https://github.com/kiwicom/margarita/issues/72
It will probably not be necessary for mobile, where you can't reach summary screen without visiting search first (and getting the data).
But on web, you can refresh the page or go directly to `/result-detail` from some shared link.

Reply:
I understand your point but I do not have a good feeling about designing schema like that, I need more think about. But maybe it is OK, let's ask for more opinion.

@michalsanger What do you think about this query?

Reply:
@FilipMessa True, I also don't have best feeling about this. In perfect world I can for example imagine to have separate query just to get itinerary data by id, but for now there is no entry point for something like this in Tequila.

Reply:
@FilipMessa Refactored, `valid` and `checked` is now directly part of the Itinerary.
Our Tequila key is working again so I was finally able to properly test it.